### DISCUSSION TITLE
Our cozy places

### DISCUSSION THREAD

**Discussion by visualapproach** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#13] [Show and tell]
You are welcome to post pictures of your SPA here for us to be inspired of. Here is my place:

![spa-2](https://user-images.githubusercontent.com/22987250/91754834-63e38e00-ebca-11ea-8861-c1cc5149666c.jpg)

**Comment by 877dev** [comment]
Very nice! My spa is down the garden while we have house renovations. I will post when it’s back in place.

**Comment by eddcaton** [comment]
![116293465_10163852282380297_9046193191604121337_o](https://user-images.githubusercontent.com/42009843/95364948-602de000-08c9-11eb-9144-e94d320c79ab.jpg)
![116221602_10163852282595297_5998162829358902489_o](https://user-images.githubusercontent.com/42009843/95364957-6328d080-08c9-11eb-9a08-fb005eb90839.jpg)

**Reply by hubaksis** [reply]
Looks nice! How do you cope with the moisture inside? 
What light did you put behind the drilled holes in the ceiling for the stars effect?

**Comment by visualapproach** [comment]
Awesome! Me like the star heaven! Thanks for showing it to us!

**Comment by eddcaton** [comment]
Thanks :) I have to say drilling almost 1000 holes by hand with a 2mm drill bit was not the highlight of my week!

**Comment by chrisblack4** [comment]
![image](https://user-images.githubusercontent.com/18100256/120940659-5ffe1280-c6ec-11eb-8f8e-58ff1ef98966.png)


I put a tv in the window too so I can watch it from inside the hot tub. The surround is insolated and I bought a hot tub cover to help keep in the heat. The other great "hack" I did was I have hose connections with shutoff values on every connection on the pump as well as the hot tub. I replaced most of the plastic hose with metal ones for washing machines and wrapped them in insolation as well. I just got my hot tub online and will be looking into getting it connected with node red automation.

**Comment by wist** [comment]
![IMG_2529](https://user-images.githubusercontent.com/4478339/121010389-a5c2e580-c795-11eb-80ae-c0f708cb4b92.jpg)
Some adjustments are still in progress but main setup is done.

**Reply by visualapproach** [reply]
Nice view!

**Comment by torei** [comment]
our 3x3 meters relax place in the garden
(just have to get rid of the cables... still..)

![pool](https://user-images.githubusercontent.com/6784876/125800190-5128ec48-6989-4903-861a-13abb05cb8ab.jpg)

**Reply by visualapproach** [reply]
Wow, that's indeed cozy! Nice lights.

**Reply by hubaksis** [reply]
Cool!
Do you have any issues with the moisture inside?

**Reply by torei** [reply]
when raining or in the winter time the air moisture is kind of high.. about 80-95%. the wood inside is impregnated so its not really an issue with water vapor at the ceiling on cold days and 40° water :-) there is no mold inside
we use a fan heater sometimes to get things more dry but most of the time we just run the fan (via openhab).

![Screenshot_20211227-144503~2](https://user-images.githubusercontent.com/6784876/147477712-499b6b0a-980a-4f5f-af66-0c744b38d742.png)

**Reply by h2owasser** [reply]
@torei how did you manage the mobile RCD (Fuse) on the Pump unit?
For me it needs to be pressed every time i disconnect power. Did you uninstall it or do you have some workaround for that as you connected main power also via smart power plug?

**Reply by torei** [reply]
hey @h2owasser.
i don't use the smart plug for my device to shut it off. just counting the used energy :-)
the pump is running all daytime and the module runs 24/7. no problems at all and i think it's using 1 W in standby (over night)

**Comment by jarisiv** [comment]
![IMG_20210919_210548](https://user-images.githubusercontent.com/68715691/147240773-413808d9-83f9-46d7-9213-656c484c3f62.jpg)

**Reply by visualapproach** [reply]
Beautiful

**Reply by jarisiv** [reply]
Thank you and thanks for the awesome project! Cheers from Finland!

**Reply by hubaksis** [reply]
Beautiful!
I assume that you have Lay-Z Helsinki judging by the wall thickness. 
I am building a deck for it right now, so looking for ideas.
If your spa is inflatable, what is the size of the hole in the deck? Does it have any vertical walls (how does it look if you take the spa out)? How do you cover it and have access to all the latches (according to the safety we need to use kids safe latches or build a fence around it)?

**Reply by jarisiv** [reply]
Thanks! Yeah it is Lay-z-spa Helsinki model. Built the surrounding / deck during last summer. There is 2 hatches; one for the pump and all the electricity and another one for all the accessories and tools etc. The round hole size is 2cm larger than the diameter of the pool. There is an 60mm insulation at the bottom and 30mm on the sides. The original cover is only temporary now. I will build a new one with insulation maybe next spring time. I will post couple pics here.

**Reply by jarisiv** [reply]
![runko](https://user-images.githubusercontent.com/68715691/147340654-cc8f243c-50c7-4991-89e4-95a59d08790b.jpg)
![eristys](https://user-images.githubusercontent.com/68715691/147340675-329d652d-40c0-4a86-9d3f-9c0425ca8b1b.jpg)
![eristys2](https://user-images.githubusercontent.com/68715691/147340684-1bdd73cf-3046-4fa3-b389-4b864e213660.jpg)
![vaiheessa](https://user-images.githubusercontent.com/68715691/147340695-36969701-6574-4baa-a25d-779f1c8971d9.jpg)
![sivu](https://user-images.githubusercontent.com/68715691/147340707-4c9c5acc-3f07-4c49-ada6-ca0b85a2fafa.jpg)
![luukut](https://user-images.githubusercontent.com/68715691/147340724-c53a572c-1a63-4646-9e98-b6a83654247b.jpg)

**Reply by jarisiv** [reply]
And at the moment looks like this! Merry Christmas everyone!
![IMG_20211224_111914](https://user-images.githubusercontent.com/68715691/147341264-1985ee38-5778-46d5-af76-fe37de1753fa.jpg)

**Reply by hubaksis** [reply]
Nice! Thank you for the details!
Merry Christmas to you too!

**Reply by torei** [reply]
nice project. very beautiful :-)

**Comment by davidmardanielsson** [comment]
Awesome spa setups. Really an inspiration. 

Mine isn't very good looking, I just added an insulating material around the tub and also inside the top, so the whole thing looks like a satellite now. But I'm planning to build a frame around it next summer. 

The insulation material is actually very effective in the top, suddenly instead of the snow melting on the top, it started to accumulate (which is a different problem)

![IMG_4943](https://user-images.githubusercontent.com/5322028/147819829-43ee4cd2-7619-4587-844b-0dee75ad8f9b.JPG)

**Reply by visualapproach** [reply]
Wow, thank you for sharing this cold paradise. 👍

**Comment by visualapproach** [comment]
@torei @davidmardanielsson @jarisiv I put some of your pictures in the build instructions. Hope you don't mind. Photo creds under the pics.

**Comment by eried** [comment]
Mine is above polar arctic circle :D so it is always very cold here

![image](https://user-images.githubusercontent.com/1091420/164948604-c523342f-5437-4e62-9385-620f60ad84ca.png)

https://www.tiktok.com/@erwinried/video/7073701043506302214?_r=1&is_copy_url=1&is_from_webapp=v1&source=h5_m

**Reply by visualapproach** [reply]
Thank you for sharing :-)
